packages/dashboard/src/components/DashboardProvider/IncomingRequest.tsx

```diff
@@ -104,12 +107,26 @@ function IncomingRequest({ provider, socket, request, setRequests }: Props) {
 
   const body = <div>{formatDashboardProviderRequestParameters(request)}</div>;
 
-  const footer = (
-    <div className="flex justify-start items-center gap-2">
-      <Button onClick={process} text="Process" />
-      <Button onClick={reject} text="Reject" />
-    </div>
-  );
+  var footer;
+  if (disable) {
+    footer = (
+      <div className="flex justify-start items-center gap-2">
+        <button
+          className="rounded p-2 text-truffle-brown uppercase hover:bg-white"
+          disabled={disable}
+        >
+          Processing...
+        </button>
```

Comment:
I noticed that this uses `<button>` whereas the existing code uses `<Button>`. Is there a way to achieve the disable behavior with `<Button>`, and could you please switch to that if so?

Reply:
I will add `disabled` to `<Button>` so that we can use it here. And add `disabled` to all places where `<Button>` is called.